@types/interfaces.d.ts

```diff
@@ -0,0 +1,54 @@
+export type TextFieldAutoCapitalize =
+  | 'on'
+  | 'off'
+  | 'none'
+  | 'characters'
+  | 'words'
+  | 'sentences';
+
+export type TextFieldAutoComplete =
```

Comment:
After looking at the spec you linked, I realized that `autocomplete` can't be a simple enum since it can have a "set of space-separated tokens" and also allows custom `section-*` token so it's practically impossible to simply list all the allowed values.

Therefore I think this should be changed back to just `{string}` so remove type `TextFieldAutoComplete`.

Reply:
Done. Removed `@type` from the `autocomplete` property for now.